src/csharp/Grpc.Core.Api/DeserializationContext.cs

```diff
@@ -48,13 +48,12 @@ public virtual byte[] PayloadAsNewBuffer()
             throw new NotImplementedException();
         }
 
-#if GRPC_CSHARP_SUPPORT_SYSTEM_MEMORY
         /// <summary>
         /// Gets the entire payload as a ReadOnlySequence.
         /// The ReadOnlySequence is only valid for the duration of the deserializer routine and the caller must not access it after the deserializer returns.
         /// Using the read only sequence is the most efficient way to access the message payload. Where possible it allows directly
         /// accessing the received payload without needing to perform any buffer copying or buffer allocations.
-        /// NOTE: This method is only available in the netstandard2.0 build of the library.
+        /// NOTE: In order to access the payload via this method, your compiler needs to support C# 7.2 (to be able to use the <c>Span</c> type).
```

Comment:
I'm a bit caught on understanding this comment - there are still non-performant ways of accessing the payload, when the Span type isn't defined, no?

Reply:
Yes, you can always access the payload through PayloadAsNewBuffer(), regardless of your framework/compiler version.
The interesting part is that once you compile the code using Span to IL, the code will work on older frameworks as well (= calling a method that internally uses Span will work everywhere, so PayloadAsNewBuffer() using PayloadAsReadOnlySequence() internally is fine), you just won't be able to reference Span<> directly in the code unless you have C# 7.2 enabled in your project. And when accessing the payload through PayloadAsReadOnlySequence(), you'd get access to the payload by referencing using the Span type directly in your code, so that's why the comment.